pkgs/by-name/re/regina-normal/package.nix

```
@@ -0,0 +1,96 @@
+{
+  lib,
+  fetchurl,
+  python3,
+  cmake,
+  cppunit,
+  doxygen,
+  gmp,
+  graphviz,
+  jansson,
+  libxml2,
+  libxslt,
+  perl,
+  pkg-config,
+  popt,
+  qt6,
+  shared-mime-info,
+  stdenv,
+  lmdb,
+  databaseLibrary ? lmdb, # can be built with tokyocabinet instead
+  highDim ? false, # whether or not to include triangulations of dimension 9-15
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "regina-normal";
+  version = "7.3.1";
+
```

```suggestion

  outputs = [
    "out"
    "dev"
    "man"
  ];

```
We might as well put development headers and manual pages in their own outputs.